app/screens/send-bitcoin-screen/send-bitcoin-confirmation-screen.tsx

```diff
@@ -231,7 +233,19 @@ const SendBitcoinConfirmationScreen = ({
     paymentAmount: paymentAmountInWalletCurrency,
   })
 
-  const feeDisplayText = fee.amount && paymentAmountToTextWithUnits(fee.amount)
+  useEffect(() => {
+    if (fee.amount) {
+      try {
+        if (paymentAmountToTextWithUnits(fee.amount) === null) {
```

Comment:
paymentAmountToTextWithUnits will never be null, so this try-catch block is unnecessary.

Reply:
> paymentAmountToTextWithUnits will never be null, so this try-catch block is unnecessary.

In the case that `fee.amount` returns an invalid value, the try catch block handles the error and records it in crashlytics. But I agree that `paymentAmountToTextWithUnits` will not return a null value, hence I'll remove the if block since its unnecessary